src/jobsautoreport/main.py

```diff
@@ -25,11 +25,14 @@ def main() -> None:
     a_week_ago = now - timedelta(weeks=1)
 
     querier = Querier(client, index_name)
-    processor = Reporter(querier)
+    reporter = Reporter(querier)
 
-    success_rate = processor.get_success_rate(a_week_ago, now)
-    number_of_jobs_triggered = processor.get_number_of_jobs_triggered(a_week_ago, now)
-    average_job_duration = processor.get_average_duration_of_jobs(a_week_ago, now)
+    success_rate = reporter.get_success_rate(a_week_ago, now)
+    number_of_jobs_triggered = reporter.get_number_of_jobs_triggered(a_week_ago, now)
+    average_jobs_duration = reporter.get_average_duration_of_jobs(a_week_ago, now)
+
+    if average_jobs_duration is None:
```

Comment:
it could be better to adapt the message instead and have a log message. Let's fix that later.